drill-admin/src/main/kotlin/com/epam/drill/endpoints/openapi/DrillAdminServer.kt

```diff
@@ -0,0 +1,122 @@
+package com.epam.drill.endpoints.openapi
+
+import com.epam.drill.common.*
+import com.epam.drill.endpoints.*
+import com.epam.drill.plugins.*
+import com.epam.drill.router.*
+import io.ktor.application.*
+import io.ktor.auth.*
+import io.ktor.http.*
+import io.ktor.http.cio.websocket.*
+import io.ktor.locations.post
+import io.ktor.response.*
+import io.ktor.routing.routing
+import org.kodein.di.*
+import org.kodein.di.generic.*
+
+class DrillAdminServer(override val kodein: Kodein) : KodeinAware {
```

Comment:
Ok, Swagger was strange... but it was autogen, what is DrillAdminServer class name mean?